src/ASM/DomainDecomposition.C

```diff
@@ -292,9 +292,9 @@ std::vector<std::set<int>> DomainDecomposition::getSubdomains(int nx, int ny, in
             continue;
 
           getSAM()->getElmEqns(eqns, el);
-          for (auto& it : eqns) {
-            if (it > 0)
-              result[d].insert(it);
+          for (auto& eqn : eqns) {
+            if (eqn > 0)
+              result[d].insert(eqn);
```

Comment:
since `eqn` is an int and is not changed, I rather prefer `for (int eqn : eqns)`here.

Reply:
done.